### ISSUE TITLE
Kokkos:  Doxygen tag file name is wrong and overwrites Tpetra's tag file

### ISSUE THREAD

**Issue by romintomasetti** [trilinos/Trilinos#10409]
## Documentation
@trilinos/\<Kokkos\>

# Description of the issue

When downloading the Tpetra Doxygen tag file from https://docs.trilinos.org/dev/packages/common/tag_files/tpetra.tag, one actually gets the Kokkos tag file content... This is clearly a bug and the line

https://github.com/trilinos/Trilinos/blob/a42024e6f161f3fd4e71a17cd79ee7bfe5ed87b6/packages/kokkos/doc/Doxyfile#L102

simply needs to be changed to

```
GENERATE_TAGFILE       = ../../common/tag_files/kokkos.tag 
```

**Comment by cgcgcg** [comment]
@trilinos/kokkos @trilinos/tpetra

**Comment by ndellingwood** [comment]
Thanks for reporting, the fix should be pulled in with PR #10253

**Comment by ndellingwood** [comment]
PR #10253 merged with the fix, closing the issue. Thanks for reporting @romintomasetti
